databases/functions/userFollowUser/check-is-blurred-post.js

```diff
@@ -13,7 +13,7 @@ module.exports = async (userFollowUserModel, userIdFollower, _transaction = null
       user_id_follower: userIdFollower,
       is_anonymous: false
     },
-    limit: MINIMUM_BLUR_USER_FOLLOWING,
+    limit: MINIMUM_BLUR_USER_FOLLOWING + 1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The limit of the database query has been increased by 1. This change could potentially alter the number of results that the query returns. If this is intended, ensure that the rest of your code can handle an extra result. If not, consider reverting this change.
```diff
-    limit: MINIMUM_BLUR_USER_FOLLOWING + 1,
+    limit: MINIMUM_BLUR_USER_FOLLOWING,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->